tests/functional/detection_status/test_detection_status.py

```diff
@@ -163,11 +168,23 @@ def test_same_file_change(self):
                     shared.ttypes.DetectionStatus.UNRESOLVED:
                 self.assertIn(report.bugHash,
                               ['cbd629ba2ee25c41cdbf5e2e336b1b1c'])
+
+                file_content = self._cc_client.getSourceFileData(
```

Comment:
should we also test for resolved bugs too?

Reply:
Done